mlflow/models/evaluation/utils/trace.py

```diff
@@ -8,11 +8,22 @@
     AUTOLOGGING_INTEGRATIONS,
     autologging_conf_lock,
     get_autolog_function,
+    is_autolog_supported,
+)
+from mlflow.utils.autologging_utils.safety import revert_patches
+from mlflow.utils.import_hooks import (
+    _post_import_hooks,
+    get_post_import_hooks,
+    register_post_import_hook,
 )
 
 _logger = logging.getLogger(__name__)
 
 
+# This flag is used to display the message only once when tracing is enabled during the evaluation.
+_IS_TRACE_MESSAGE_DISPLAYED = False
```

Comment:
```suggestion
_SHOWN_TRACE_MESSAGE_BEFORE = False
```

Can we rename this flag?